src/migrations/20181211215451-ArticleTag.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('ArticleTag', {
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    ArticleId: {
```

Comment:
I would suggest you change this to camelCase as stated in the team's convention. Also, change it for all other instances.
```suggestion
    articleId: {
```

Reply:
Thank you, will fix the naming of the articleId

Reply:
This chore is a follow-up to the query article feature I am working on. `Article should be queried by author and tags.`
Based on my interaction with the TTL, he asked me to raise a PR of create tag model to the query article branch and that is why I have raised this PR to the user-search-article-functionality-161779793 branch instead of develop.

Reply:
Okay 👍